cvat-ui/src/components/analytics-page/task-consensus/task-consensus-component.tsx

```diff
@@ -0,0 +1,221 @@
+                });
+            }
+        }
+
+        core.consensus
+            .reports({
+                pageSize: 1, target: 'task', jobID: null, taskID: task.id,
+            })
+            .then(([report]) => {
+                let reportRequest = Promise.resolve<ConsensusReport[]>([]);
+                let assigneeReportRequest = Promise.resolve<AssigneeConsensusReport[]>([]);
+                if (report) {
+                    reportRequest = core.consensus.reports({
+                        pageSize: task.jobs.length,
+                        taskID: task.id,
+                        target: 'job',
+                    });
+                    assigneeReportRequest = core.consensus.assignee_reports({
+                        taskID: task.id,
+                        consensusReportID: report.id,
+                    });
+                }
+
+                Promise.all([reportRequest])
+                    .then(([jobReports]) => {
+                        dispatch(reducerActions.setTaskReport(report || null));
+                        dispatch(reducerActions.setJobsReports(jobReports));
+                        Promise.all([assigneeReportRequest])
+                            .then(([assigneeReports]) => {
+                                dispatch(reducerActions.setAssigneeReports(assigneeReports));
+                            });
+                    })
+                    .catch(handleError)
+                    .finally(() => {
+                        dispatch(reducerActions.setFetching(false));
+                    });
+            })
+            .catch(handleError);
+    }, [task?.id]);
+
+    const {
+        fetching, taskReport, jobsReports, assigneeReports,
+    } = state;
+
+    const onTabKeyChange = useCallback((key: string): void => {
+        setTab(key as DetailsTabs);
+    }, []);
+
+    tabs = (
+        <Tabs
+            type='card'
+            activeKey={activeTab}
+            defaultActiveKey={DetailsTabs.JOBS}
+            onChange={onTabKeyChange}
+            className='cvat-task-analytics-tabs'
+            items={[
+                {
+                    key: DetailsTabs.JOBS,
+                    label: 'Jobs',
+                    children: <JobList jobsReports={jobsReports} task={task} />,
+                },
+                {
+                    key: DetailsTabs.ASSIGNEES,
+                    label: 'Assignees',
+                    children: <AssigneeListComponent assigneeReports={assigneeReports} task={task} />,
+                },
+            ]}
+        />
+    );
+
+    return (
+        <div className='cvat-task-quality-page'>
+            {fetching ? (
+                <CVATLoadingSpinner size='large' />
+            ) : (
+                <>
+                    <Row>
+                        <Text type='secondary'>
+                            {`Created ${taskReport?.id ? moment(taskReport.createdDate).fromNow() : ''}`}
```

Comment:
When we dont have a report it just says `Created` and empty space. Lets show `Created` only when we have the report